data/ADITYADAS1999.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "ADITYA DAS",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open source enthusiast",
+  "avatar": "https://github.com/ADITYADAS1999.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ADITYA90546170",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
+      "url": "https://www.linkedin.com/in/aditya-das-7b2276202",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "visit my portfollio",
+      "url": "https://adityadas1999.github.io/",
+      "icon": "FaGlobe"
+    },
+    {
+      "name": "Follow me on Youtube",
+      "url": "https://www.youtube.com/channel/UCUBqrPu1GoR0Cfgl5rDHIbg",
+      "icon": "FaYoutube"
+    },
+    {
+      "name": "visit my Github profile",
+      "url": "https://adityadas1999.github.io/",
+      "icon": "FaGithub"
+    }
+  ]
+}
+
+
+
```

Comment:
There are three extra lines in here.
```suggestion
```

Reply:
> There are three extra lines in here.

Thanks for reviewing the PR [Panquesito7](https://github.com/Panquesito7) 😃

Reply:
We have Prettier GitHub Action that should fix these formatting issues (if the json is valid), so not a blocker